napari/layers/image/image.py

```diff
@@ -533,8 +533,8 @@ def _update_thumbnail(self):
             color_range = high - low
             if color_range != 0:
                 downsampled = (downsampled - low) / color_range
-            colormapped = self.colormap[1].map(downsampled)
-            colormapped = colormapped.reshape(downsampled.shape + (4,))
+            color_array = self.colormap[1][downsampled.ravel()]
+            colormapped = color_array.rgba.reshape(downsampled.shape + (4,))
```

Comment:
why lowercase here and uppercase elsewhere? (I prefer lowercase if they are equivalent)

Reply:
That’s just the [ColorArray API](http://vispy.org/color.html#vispy.color.ColorArray).
Uppercase gives you a unit8 array of [0, 255], lowercase gives you float of [0, 1].